engine/src/main/java/org/terasology/engine/bootstrap/EntitySystemSetupUtil.java

```diff
@@ -42,15 +42,7 @@
 import org.terasology.network.NetworkSystem;
 import org.terasology.persistence.typeHandling.TypeSerializationLibrary;
 import org.terasology.persistence.typeHandling.extensionTypes.EntityRefTypeHandler;
-import org.terasology.recording.CharacterStateEventPositionMap;
-import org.terasology.recording.DirectionAndOriginPosRecorderList;
-import org.terasology.recording.EntityIdMap;
-import org.terasology.recording.EventCatcher;
-import org.terasology.recording.RecordAndReplayUtils;
-import org.terasology.recording.RecordAndReplayStatus;
-import org.terasology.recording.EventSystemReplayImpl;
-import org.terasology.recording.RecordAndReplaySerializer;
-import org.terasology.recording.RecordedEventStore;
+import org.terasology.recording.*;
```

Comment:
Wildcard import - check on your IDE settings to make them not collapse please :-)